packages/mobile/src/components/lineup-tile/LineupTileStats.tsx

```diff
@@ -205,24 +192,14 @@ export const LineupTileStats = ({
         </View>
       )}
       {premiumConditions && !isOwner ? (
-        <View
-          style={[
-            styles.iconLocked,
-            styles.listenCount,
-            doesUserHaveAccess ? styles.iconUnlocked : null,
-            isUSDCEnabled &&
-            isPremiumContentUSDCPurchaseGated(premiumConditions) &&
-            doesUserHaveAccess
-              ? styles.iconUSDC
-              : null
-          ]}
-        >
-          {doesUserHaveAccess ? (
-            <IconUnlocked fill={staticWhite} height={14} width={14} />
-          ) : (
-            <IconLock fill={staticWhite} height={14} width={14} />
-          )}
-        </View>
+        <LockedStatusBadge
+          locked={!doesUserHaveAccess ?? true}
```

Comment:
?

Reply:
`doesUserHaveAccess` could be undefined. default to true seems better than false?

Reply:
! is going to convert the value to a boolean. Anything falsey (undefined, null, 0, an empty string) will result in `locked` being `true`. So I think null coalescing to `true` isn't necessary here?
At first glance, I thought this was a temporary thing you were doing for testing.